src/VisualStudio/VisualStudioDiagnosticsToolWindow/Loggers/OutputWindowLogger.cs

```diff
@@ -32,6 +35,18 @@ public OutputWindowLogger(IGlobalOptionService optionService)
         public OutputWindowLogger(Func<FunctionId, bool> loggingChecker)
         {
             _loggingChecker = loggingChecker;
+
+            var serviceProvider = ServiceProvider.GlobalProvider;
+
+            var componentModel = (IComponentModel)serviceProvider.GetService(typeof(SComponentModel));
+            Assumes.Present(componentModel);
+
+            var brokeredServiceContainer = (IBrokeredServiceContainer)serviceProvider.GetService(typeof(SVsBrokeredServiceContainer));
```

Comment:
📝 This should be using `GetServiceAsync`, and provided as a constructor argument